tools/clang/test/DXC/fd_zs.test

```diff
@@ -0,0 +1,7 @@
+// /Fd plus /Zs
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zs /Fd %t.pdb /Fo %t.dxo_fd
+// RUN:FileCheck --input-file=%t.pdb %s --check-prefix=PDB
+// Make sure PDB not empty.
+// PDB:{{.+}}
+// RUN:FileCheck --input-file=%t.dxo_fd %s --check-prefix=FD_FO
```

Comment:
My feeling is temp file not being deleted, but not ready.
Move File check in another line will make the file ready.

Reply:
I'm not sure I understand how that would be possible. Since these are each different processes, each process starts after the last one completes. The filesystem updates should be flushed before the the next process starts.

I wonder if there is something else going on here that might be a bug we should investigate.

Reply:
I share that concern

Reply:
Upstream test has some pattern.
Like https://github.com/llvm/llvm-project/blob/main/llvm/test/Analysis/Dominators/print-dot-dom.ll#L2

Reply:
This pattern here is fine, but the test case with the `extra_output` above still seems wrong. We should be able to have those on separate `RUN` lines.

Reply:
Could you show what you want with example?
Is the // PDB:{.+} you don't like?